step-certificates/templates/ingress.yaml

```
@@ -36,8 +36,10 @@ spec:
           {{- range .paths }}
           - path: {{ . }}
             backend:
-              serviceName: {{ $fullName }}
-              servicePort: {{ $svcPort }}
+              service:
+                name: {{ $fullName }}
+                port:
+                  number: {{ $svcPort }}
```

Does this depend on the apiVersion? This is ok for `networking.k8s.io/v1` But apparently on older versions it was `serviceName`, `servicePort` https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122

Can you add a conditional like this one:
```suggestion
{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
              service:
                name: {{ $fullName }}
                port:
                  number: {{ $svcPort }}
{{- else }}
              serviceName: {{ $fullName }}
              servicePort: {{ $svcPort }}
{{- end }}
```